code/go/internal/validator/folder_item_spec_errors.go

```diff
@@ -4,11 +4,15 @@
 
 package validator
 
-import "github.com/elastic/package-spec/code/go/internal/validator/semantic"
+import (
+	"fmt"
+
+	"github.com/elastic/package-spec/code/go/internal/validator/semantic"
+)
 
 func adjustErrorDescription(description string) string {
 	if description == "Does not match format '"+semantic.RelativePathFormat+"'" {
-		return "relative path is invalid or target doesn't exist"
+		return fmt.Sprintf("relative path is invalid, target doesn't exist or is bigger than %s", semantic.RelativePathFileMaxSize)
 	} else if description == "Does not match format '"+semantic.DataStreamNameFormat+"'" {
 		return "data stream doesn't exist"
 	}
```

Comment:
Avoiding the need of these adjustments would be another reason to look for another jsonschema library with better customization options.